README.md

```diff
@@ -46,6 +46,9 @@ Adaptive course authoring platform.
   enter configuration details.
 - Initialize database by running `npm run db migrate`
 - Enable default repository schema by setting env variable `ENABLE_DEFAULT_SCHEMA=1` or define a custom one (more details below).
+- Configure Tailor by following the steps [in the configuration section](#content-repository-structure),
+specifically creating a `tailor.config.js` out of `tailor.config.js.example`.
```

Comment:
Please align the indentation on multi-row bullets the same way it is done on lines 45/46 💅